composer/loggers/remote_uploader_downloader.py

```diff
@@ -671,6 +671,10 @@ def _upload_worker(
                 continue
         uri = remote_backend.get_uri(remote_file_name)
 
+        local_rank = dist.get_local_rank()
+        local_rank_stagger = int(os.environ.get('COMPOSER_LOCAL_RANK_STAGGER_SECONDS', 0))
```

Comment:
Can you also add a debug log here

Reply:
done (although i think this might get swallowed by the subprocess)